src/utils/transaction.ts

```diff
@@ -717,49 +628,30 @@ export const buildCrossChainAction = async (
             amount: amountBN.toString(),
             iconUrl: fromAssetIconUrl,
           },
-          amount: BigNumber.from(routeData?.bestRoute?.toAmount),
+          amount: 1,
           toAsset: {
             address: plrDaoAsset.address,
-            decimals: 9,
-            symbol: 'PillarDAO',
-            amount: result.result?.returnAmount.toString() ?? '0',
+            decimals: toAssetDecimals,
+            symbol: 'Pillar DAO NFT',
+            amount: '1',
             iconUrl: 'https://public.pillar.fi/files/pillar-dao-member-badge.png',
           },
           receiverAddress: transactionBlock?.values?.receiverAddress,
-          providerName: result.result?.provider
-            ? swapServiceIdToDetails[result.result.provider].title
-            : 'Unknown provider',
-          providerIconUrl: result.result?.provider ? swapServiceIdToDetails[result.result.provider].iconUrl : '',
+          providerName: '',
+          providerIconUrl: '',
```

Comment:
remove?

Reply:
It is a required field as other blocks are using this fields in the preview.

Reply:
it is not required field by other blocks, please check your block type and apply changes accordingly

Reply:
@parag-chauhan this is not fixed